clang/include/clang/Sema/DeclSpec.h

```diff
@@ -2325,13 +2328,16 @@ class Declarator {
   /// This function takes attrs by R-Value reference because it takes ownership
   /// of those attributes from the parameter.
   void AddTypeInfo(const DeclaratorChunk &TI, ParsedAttributes &&attrs,
-                   SourceLocation EndLoc) {
+                   SourceLocation EndLoc,
+                   const DeclaratorChunk::LateAttrListTy &LateAttrs = {}) {
```

Comment:
This might actually be fine, and I haven't thought through the implications yet, but `SmallVectorImpl<T>&` is generally used in the parameter type, rather than `SmallVector<T, N>&`. Do we want to use the concrete type here?

Reply:
Good question. I used `SmallVector<T, N>&` to allow default initialization. I considered creating an overload that takes `SmallVectorImpl<T>&`, but a single function with a default-initialized parameter felt simpler. That said, I'm open to other suggestions.

Reply:
How about `ArrayRef`? It has the benefit of not needing the `N`, and an empty ArrayRef can be used as default argument.